tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -399,6 +399,12 @@ constexpr bool test_input_range() {
     STATIC_ASSERT(std::same_as<ranges::range_rvalue_reference_t<Range&>, RvalueReference>);
     STATIC_ASSERT(std::same_as<ranges::range_rvalue_reference_t<Range&&>, RvalueReference>);
 
+    // LWG-3860, validate ranges::range_common_reference_t
+    using RcommonReference = std::iter_common_reference_t<Iterator>;
+    STATIC_ASSERT(std::same_as<ranges::range_common_reference_t<Range>, RcommonReference>);
+    STATIC_ASSERT(std::same_as<ranges::range_common_reference_t<Range&>, RcommonReference>);
+    STATIC_ASSERT(std::same_as<ranges::range_common_reference_t<Range&&>, RcommonReference>);

```

Comment:
Why "**R**commonReference" and not just "CommonReference"?

```suggestion
    using CommonReference = std::iter_common_reference_t<Iterator>;
    STATIC_ASSERT(std::same_as<ranges::range_common_reference_t<Range>, CommonReference>);
    STATIC_ASSERT(std::same_as<ranges::range_common_reference_t<Range&>, CommonReference>);
    STATIC_ASSERT(std::same_as<ranges::range_common_reference_t<Range&&>, CommonReference>);
```